README.md

```diff
@@ -3,7 +3,10 @@
 package_updater
 ===========
 
-Basic description for package_updater
+Run the ansible package module to install, upgrade or remove
+packages using the underlying OS package manager of the target system.  This role
+will work with a variety of package managers like; yum, dnf, and apt that support
```

Comment:
Don't name it if we don't test it - e.g "apt". If you're going to name it, add a test scenario for it.

Reply:
README.md redone